tests/skyserve/spot/spot_recovery.yaml

```
@@ -0,0 +1,16 @@
+resources:
+  cloud: gcp
+  cpus: 2+
+  use_spot: true
```

```suggestion
  zone: us-central1-a
  use_spot: True
```
Add zone for the smoke test gcloud delete cmd; also nit to keep align with non_recovery.yaml